spec/models/project_spec.rb

```diff
@@ -7,4 +7,21 @@
     it {should have_many :contestant_projects}
     it {should have_many(:contestants).through(:contestant_projects)}
   end
+
+  describe "#num_contestants" do
+    it "returns number of contestants a project has" do
+      @recycled_material_challenge = Challenge.create(theme: "Recycled Material", project_budget: 1000)
+
+      @news_chic = @recycled_material_challenge.projects.create(name: "News Chic", material: "Newspaper")
+
+      @jay = Contestant.create(name: "Jay McCarroll", age: 40, hometown: "LA", years_of_experience: 13)
+      @gretchen = Contestant.create(name: "Gretchen Jones", age: 36, hometown: "NYC", years_of_experience: 12)
+
+      ContestantProject.create(contestant_id: @gretchen.id, project_id: @news_chic.id)
+      ContestantProject.create(contestant_id: @jay.id, project_id: @news_chic.id)
+
+
+      expect(@news_chic.num_contestants).to eq(2)
+    end
+  end
```

Comment:
A rule of thumb, when doing instance method tests, always test more than one instance. Currently, just looking at your assertion on line 24, there is no wat to definitively say that your `num_contestants` method isn't just returning a hard-coded `2`. By making a call on another instance, it gives you the ability to return a different value and prove that the code isn't hard-coded.